api/lib/cors.js

```diff
@@ -0,0 +1,34 @@
+const Config = require("./lib/config");
+const Cors = require("cors");
+
+module.exports = function({ type = "default" }) {
+    const whitelist = Config.defaultAllowedOrigins;
+
+    if (type === "public") {
```

Comment:
Just export a publicCors object from this file. In this way, we don't have to pass configuration options.